psd-web/spec/models/investigation_spec.rb

```diff
@@ -78,4 +78,30 @@
       end
     end
   end
+
+  describe "ownership" do
+    let(:user)          { create(:user, :activated, has_viewed_introduction: true) }
+    let(:investigation) { create(:investigation, creator: user) }
```

Comment:
The owner defaults to the creator by the factory if not set. Since we're testing the owner behaviour here we should set it explicitly:

```suggestion
    let(:investigation) { create(:investigation, owner: user, creator: user) }
    let(:owner) { user }
```
We can then override `let(:owner)` in the following `context` blocks.

Reply:
I am testing setter here, so I would prefer to do it explicitly.